modules/apps/frontend-js/frontend-js-module-launcher/src/main/resources/META-INF/resources/webpack_federation.js

```diff
@@ -334,7 +356,12 @@
 	 * Split a module name in its `containerId` and `path` parts
 	 */
 	function splitModuleName(moduleName) {
-		const i = moduleName.indexOf('/');
+		let i = moduleName.indexOf('/');
+
+		// Skip first / for scoped packages
```

Comment:
I expect SF to complain about this line.

Reply:
I ran SF for that module and it didn't complain :thinking: . Looked strange to me, that's why I wanted to see what the CI said. But since you think that SF is bad too, I'll have a look again.

Reply:
> If you're baby-sitting this for days before actually sending

I don't know. Are we going to send this or do we need to change all the packages at once? I don't know what you agreed with Brian (if anything) or what limitations/considerations we must take into account.

If it was for me, I would send just this and then continue, I guess...

Reply:
Also, we cannot send this until [this](https://github.com/liferay/liferay-frontend-projects/pull/408) gets merged and released :grimacing:

Reply:
> If it was for me, I would send just this and then continue, I guess...

Yeah, I would send this first, because these are the only actually problematic packages at this time.